src/main/java/steps/AndroidLoginSteps.java

```diff
@@ -0,0 +1,56 @@
+package steps;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.By;
+import skynet.runner.WebDriverManager;
+import skynet.utils.Logger;
+
+public class AndroidLoginSteps {
+    @When("user tap on the Login text button")
+    public void tapOnLoginButton() throws Exception {
+        WebDriverManager.getAppiumDriver().findElement(By.id("textViewLogin")).click();
+        Logger.info("should see correct Login text button");
+    }
+
+    @And("user select and tap on Germany")
+    public void selectAndTapOnDropDown() throws Exception {
+        Logger.info("click on the drop-down button");
+        WebDriverManager.getAppiumDriver().findElement(By.xpath("//android.widget.TextView[contains(@text,'Germany')]")).click();
+        Thread.sleep(2000);
+    }
+
+    @And("user enter the username {string}")
+    public void userEnterUsername(String email) {
+        Logger.info("enter username");
+        WebDriverManager.getAppiumDriver().findElement(By.id("editTextEmail")).sendKeys(email);
```

Comment:
As you are already aware we do not use this king of syntax with our tests for reasons we are aware of already. If there's a need to identify elements using a specific driver such as `getAppiumDriver()` let's treat that in the framework

Reply:
step defs is updated